src/windows/MediaProxy.js

```diff
@@ -35,6 +46,8 @@ module.exports = {
         var src = args[1];
         var thisM = Media.get(id);
 
+        src = cdvfileToNative(src);
```

Comment:
Yes, thanks!
Although I think of moving this commit to [CB-9238 Media plugin cannot record audio on windows](https://github.com/apache/cordova-plugin-media/pull/58) PR as it requires paths handling introduced there.